src/system/ThemeModule/composer.json

```diff
@@ -15,7 +15,6 @@
     },
     "require": {
         "php": ">=7.2.0",
-        "twig/twig": "2.*",
```

Comment:
why move this to CoreBundle?

Reply:
Because our Twig requirement is not theme-specific but general.

Reply:
Well, you could almost make that argument about most vendors. for the record, the ThemeModule is also not theme specific, but general. It was placed here because this is where the Twig\Environment override is located. But I'm fine with it either way I guess.